development/gardener-rotate/README.md

```diff
@@ -0,0 +1,52 @@
+# gardener-rotate
+
+## Overview
+
+gardener-rotate tool allows to generate a new access token for Gardener service accounts and update kubeconfig stored in the Secret Manager.
+
+Rotation process steps:
```

Comment:
It looks like the formatting is not as expected. Maybe you can use a, b, c for the sub-sections? 
![image](https://user-images.githubusercontent.com/62639546/185362691-593bd8e4-7086-4895-bfee-1a5a5372a5a3.png)

Reply:
Indents should be 4 spaces instead of 2, now three points should be indented.